Examples/Python/python/acts/examples/itk.py

```diff
@@ -439,6 +440,17 @@ def itkSeedingAlgConfig(inputSpacePointsType: InputSpacePointsType):
         maxSeedsPerSpMConf = 5
         maxQualitySeedsPerSpMConf = 5
         useDeltaRorTopRadius = True
+
+        if fastSeeding == True:
```

Comment:
We need to make sure all these cuts are just configuration options and that `fast` is not used in the name of the variables. `fastSeeding` is very much misleading, as this is ATLAS jargon (and also within ATLAS we need to clarify what we mean with it)

Reply:
agree. also, `fast` would imply we are committed to never write a faster code in the future